prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -292,3 +294,134 @@ func TestHandle(t *testing.T) {
 		}
 	}
 }
+
+func TestParseOwnersFile(t *testing.T) {
+	cases := []struct {
+		name  string
+		patch string
+		err   bool
+		line  int
+	}{
+		{
+			name: "emptyApprovers",
```

Comment:
I assume this is causing some magic to load an emptyApprovers document? I would prefer we inline the document, so I don't have to understand how we fetch it.

Reply:
It's using the zero value in the struct. but I agree